### PR TITLE
docs: add "阿里云 RDS 支持限制"

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What problem does this PR solve? 

- add a doc about "阿里云 RDS 支持限制"

### What is changed and how it works?

- add a doc and link to it

### Check List 

Tests 

 - No code

### CODE REVIEW

**Comment by csuzhangxc** [comment]
@GregoryIan @amyangfei PTAL

File: docs/dm/zh_CN/vendors/aliyun-rds.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+当打开阿里云 RDS 特有的 `implict_primary_key` 参数后，如果用户创建的表中没有指定主键或唯一键，RDS 会自动为表创建隐式主键。
+
+当使用 DM 从包含隐式主键的表增量同步数据到下游 TiDB 时，会由于 binlog event 中包含隐式主键数据而造成与下游表结构的不匹配，导致数据同步中断。
+
+#### 解决办法
+
+1. 对于将要创建的表，在创建表时显式地指定主键
+2. 对于已经存在且包含隐式主键的表，在关闭 `implict_primary_key` 参数后，显式地为表增加主键
+
+### 缺少 SUPER privilege(s) 权限 dump 失败
```
Comment (kennytm) [inline docs/dm/zh_CN/vendors/aliyun-rds.md]:
```suggestion
### 缺少 SUPER 权限造成 dump 失败
```

Privilege already means 权限

**Review by amyangfei — COMMENTED** [review]
LGTM

**Comment by csuzhangxc** [comment]
@kennytm PTAL again

File: docs/dm/zh_CN/vendors/aliyun-rds.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+- [implict_primary_key 造成同步中断](#implict_primary_key-造成同步中断)
+- [缺少 SUPER privilege(s) 权限 dump 失败](#缺少-super-privileges-权限-dump-失败)
+
+> 注意：目前 DM 未针对阿里云 RDS 进行完备的测试，除以下已列出的功能在支持上受限外，可能还存在其他功能无法完整支持，如需在生产环境中使用 DM，请先根据实际业务场景进行完备的测试。
+
+### implict_primary_key 造成同步中断
+
+当打开阿里云 RDS 特有的 `implict_primary_key` 参数后，如果用户创建的表中没有指定主键或唯一键，RDS 会自动为表创建隐式主键。
+
+当使用 DM 从包含隐式主键的表增量同步数据到下游 TiDB 时，会由于 binlog event 中包含隐式主键数据而造成与下游表结构的不匹配，导致数据同步中断。
```
Comment (IANTHEREAL) [inline docs/dm/zh_CN/vendors/aliyun-rds.md:15]: 此外TiDB 不支持 RDS 的 `implict_primary_key`，并且 dm 对不存在主键或者唯一索引的表的同步不保证正确性，建议对所有的表加上主键

**Comment by IANTHEREAL** [comment]
Rest LGTM

**Comment by IANTHEREAL** [comment]
LGTM

File: docs/dm/zh_CN/vendors/aliyun-rds.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+阿里云 RDS 支持限制
+===
+
+### 内容索引
+
+- [implict_primary_key 造成同步中断](#implict_primary_key-造成同步中断)
+- [缺少 SUPER privilege(s) 权限 dump 失败](#缺少-super-privileges-权限-dump-失败)
```
Comment (kennytm) [inline docs/dm/zh_CN/vendors/aliyun-rds.md:7]: Update this link

**Review by kennytm — COMMENTED** [review]
Rest LGTM

File: docs/dm/zh_CN/vendors/aliyun-rds.md (hunk 3 above)
Comment (csuzhangxc) [reply docs/dm/zh_CN/vendors/aliyun-rds.md:7]: :anguished: already merged.

Comment (csuzhangxc) [reply docs/dm/zh_CN/vendors/aliyun-rds.md:7]: I'll push a commit to the doc branch directly to update it.

Comment (kennytm) [reply docs/dm/zh_CN/vendors/aliyun-rds.md:7]: Ok 🤣

**Pull request merged**